### ISSUE TITLE
go dep is not supported well

### ISSUE THREAD

**Issue by denverdino** [aliyun/alibaba-cloud-sdk-go#272]
* **Product and API**:
* **Platform**:
* **How to Reproduce**

```
$ dep ensure --add github.com/aliyun/alibaba-cloud-sdk-go@^1.60.4
Fetching sources...

Solving failure: No versions of github.com/aliyun/alibaba-cloud-sdk-go met constraints:
	1.27.7: Could not introduce github.com/aliyun/alibaba-cloud-sdk-go@1.27.7, as it is not allowed by constraint ^1.60.4 from project github.com/heptio/velero.
	1.60.5: Could not introduce github.com/aliyun/alibaba-cloud-sdk-go@1.60.5, as its subpackage github.com/aliyun/alibaba-cloud-sdk-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	1.60.4: Could not introduce github.com/aliyun/alibaba-cloud-sdk-go@1.60.4, as its subpackage github.com/aliyun/alibaba-cloud-sdk-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	1.60.3: Could not introduce github.com/aliyun/alibaba-cloud-sdk-go@1.60.3, as it is not allowed by constraint ^1.60.4 from project github.com/heptio/velero.
	1.60.2: Could not introduce github.com/aliyun/alibaba-cloud-sdk-go@1.60.2, as it is not allowed by constraint ^1.60.4 from project github.com/heptio/velero.
```

And

```
$ dep ensure --add github.com/aliyun/alibaba-cloud-sdk-go/services/ecs@^1.60.4
Fetching sources...

Solving failure: No versions of github.com/aliyun/alibaba-cloud-sdk-go met constraints:
	1.27.7: Could not introduce github.com/aliyun/alibaba-cloud-sdk-go@1.27.7, as it is not allowed by constraint ^1.60.4 from project github.com/heptio/velero.
	1.60.5: Could not introduce github.com/aliyun/alibaba-cloud-sdk-go@1.60.5, as it has a dependency on github.com/jmespath/go-jmespath with constraint ^0.2.2, which has no overlap with existing constraint 0b12d6b5 from github.com/aws/aws-sdk-go@v1.16.0
	1.60.4: Could not introduce github.com/aliyun/alibaba-cloud-sdk-go@1.60.4, as it has a dependency on github.com/jmespath/go-jmespath with constraint ^0.2.2, which has no overlap with existing constraint 0b12d6b5 from github.com/aws/aws-sdk-go@v1.16.0

```

That is very frustrating experience

It is dup for #93, and we should not close it for work as design

**Comment by denverdino** [comment]
We can just add one empty .go file to make it work. Thanks

**Comment by JacksonTian** [comment]
@denverdino We will resolve it asap. Thank you very much.

**Comment by wenzuochao** [comment]
> We can just add one empty .go file to make it work. Thanks

We have fixed it at pr #273 . Hope it can help you. Could you try again?

**Comment by wenzuochao** [comment]
> > We can just add one empty .go file to make it work. Thanks
> 
> We have fixed it at pr #273 . Hope it can help you. Could you try again?

Have you solved your problem？I hope you can give me an answer， or I will close this issue in two days. Thanks for your support for our project.

**Comment by denverdino** [comment]
Thanks
